test/1_stdlib/SetAlgebra.swift

```diff
@@ -0,0 +1,188 @@
+// RUN: %target-run-simple-swift
+// REQUIRES: executable_test
+
+import StdlibUnittest
+
+public struct DefaultedSetAlgebra<Element> : SetAlgebra {
+  public typealias Element = String
```

Comment:
This typealias breaks the generic nature of this type.  I think you should remove it.

Instead, you should add the `Element : Hashable` constraint so that you can create a `Set<Element>` as `var store`.